runtime/litmus/src/tests/xcm_parachain/mod.rs

```diff
@@ -14,1148 +14,96 @@
 // You should have received a copy of the GNU General Public License
 // along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
 
-use super::setup::*;
```

Comment:
I believe the whole file can be renamed to `xcm_parachain.rs` to reduce confusion (we don't need an extra mod.rs here)

Reply:
Renamed this file to `xcm_parachain.rs`